src/store.js

```diff
@@ -52,6 +52,10 @@ export const selectors = {
     return isUpdating(state.openmrs.patients);
   },
 
+  isPatientStoreInErrorState: (state) => {
+    return isError(state.openmrs.patients)
```

Comment:
Why are we making it so that patients might sometimes be a set of patients or sometimes be an error object? Rather than just making an independent error object in the store?

Reply:
This is an independent error boolean (though maybe it should be object)... you pass in the "openmrs.patients" section of the store, and then isError (defined in the patient reducer) pulls off the error boolean:

https://github.com/openmrs/openmrs-react-components/pull/234/files#diff-92b696be9aefe0eeef32fb515a0431a7R185

(This was a pattern I saw somewhere... reducers define selectors for their sections of the store, and then they are all pulled together and exposed for outside use in store.js, willing to consider something else if this is confusing)